src/sdk/data/data.module.ts

```diff
@@ -249,4 +249,65 @@ export class DataModule {
       throw new Error(error.message || 'Failed to fetch exchange rates');
     }
   }
+
+  async getConnextSupportedAssets(chainId?: number): Promise<ConnextToken[]> {
+    try {
+      const queryParams = {
+        'api-key': this.currentApi,
+        chainId,
+      };
+
+      const result = await this.apiService.makeRequest(API_ENDPOINTS.GET_CONNEXT_SUPPORTED_ASSETS, MethodTypes.GET, queryParams);
+
+      return result ? result.tokens : [];
+    } catch (error) {
+      throw new Error(error.message || 'Failed to get connext supported assets');
+    }
+  }
+
+  async getConnextQuotes(
+    fromAddress: string,
+    toAddress: string,
+    fromChainId: number,
+    toChainId: number,
+    fromToken: string,
+    fromAmount: BigNumber,
+    slippage: number
+  ): Promise<ConnextTransaction[]> {
+    try {
+      const queryParams = {
+        'api-key': this.currentApi,
+        fromAddress,
+        toAddress,
+        fromChainId,
+        toChainId,
+        fromToken,
+        fromAmount: fromAmount.toString(),
+        slippage
+      };
+
+      const result = await this.apiService.makeRequest(API_ENDPOINTS.GET_CONNEXT_QUOTE_TRANSACTIONS, MethodTypes.GET, queryParams);
+
+      return result ? result.transactions : [];
+    } catch (error) {
+      throw new Error(error.message || 'Failed to get connext quotes transactions');
+    }
+  }
+
+  async getConnextTransactionStatus(fromChainId: number, toChainId: number, transactionHash: string): Promise<ConnextTransactionStatus> {
+    try {
+      const queryParams = {
+        'api-key': this.currentApi,
```

Comment:
Again. Why do we see this discrepancy everywhere? Can we stick to CamelCase throughout?

Reply:
The `api-key` parameter is set by **TYK** to authenticate the user.